_posts/2015-12-14-05-Time-Series-Raster-In-R.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: post
```

Comment:
was the name of this file changed as well? i can't see the changes. it looks like a new file in the PR